proto/bootz.proto

```diff
@@ -103,42 +103,43 @@ message ControlCardState {
     CONTROL_CARD_STATUS_INITIALIZED = 2; 
   }
 
+  // Serial must align with the serial numberpre
   string serial_number = 1;
   ControlCardStatus status = 2;
 }
 
-message GetBootstrapDataResponse {
+message BootstrapDataResponse {
```

Comment:
What's our expectation on stability of the API at this point? I presume we're OK with making backwards incompatible changes since this is still early?

Reply:
for now